docs/authentication-backends.md

```diff
@@ -1,44 +1,40 @@
 # Authentication & Backends
 
-Netlify CMS stores content in your GitHub repository (GitLab and Bitbucket coming soon!). You'll
-need to sign in to your GitHub account through the CMS for this to work, and it requires a server.
-We have a few options for handling this.
+Netlify CMS stores content in your GitHub repository. (GitLab and Bitbucket coming soon!) In order for this to work, you need to authenticate with GitHub, and that requires a server. We have a few options for handling this.
 
 ## Git Gateway with Netlify Identity
 
-[Git Gateway](https://github.com/netlify/git-gateway) another Netlify open source project, together
-with the [Netlify Identity](https://www.netlify.com/docs/identity/) service, provides a way for CMS
-users to sign in with email and password, and doesn't require them to have a GitHub account. This is
-a great option for non-technical editors, or if you don't want CMS users to
-have direct access to your GitHub repo. You can try Git Gateway with Netlify Identity any time via
-the [Test Drive](/test-drive/).
+[Git Gateway](https://github.com/netlify/git-gateway) is a Netlify open source project that allows you to add editors to your site CMS without giving them direct push access to your GitHub repository. [Netlify Identity](https://www.netlify.com/docs/identity/) service handles the authentication and provides a simple interface for user management. The Netlify CMS [Test Drive](/test-drive/) is a working example of this backend.
 
-Using it in your own project is simple:
+To use it in your own project, follow these steps:
```

Comment:
I try to avoid using words like "simple," "easy," or "just do x" in docs, since the reader may disagree and/or feel judged.